docs/configuration.md

```diff
@@ -804,7 +804,7 @@ Apart from these, the following properties are also available, and may be useful
   <td>
     Show the progress bar in the console. The progress bar shows the progress of stages
     that run for longer than 500ms. If multiple stages run at the same time, multiple
-    progress bars will be displayed on the same line.
+    progress bars will be displayed on the same line. This is applied only in shells.
```

Comment:
It will still be enabled for normal apps if explicitly set in the configuration, so your update is not really correct.